great_expectations/experimental/datasources/pandas_datasource.py

```diff
@@ -135,7 +135,19 @@ def get_batch_list_from_batch_request(
         for request, path in self._fully_specified_batch_requests_with_path(
             batch_request
         ):
-            batch_spec = PathBatchSpec(path=str(path))
+            batch_spec = PathBatchSpec(
```

Comment:
@Kilo59 @billdirks It looks like the direction is toward `DataFrameDataAsset` rather than `FilesystemDataAsset` -- in other words, the focus on the computation (e.g., `DataFrame`), not the source of data (e.g., local filesystem, or cloud filesystem, etc.).  I am curious about the following questions:
* How would we represent an `SQL` data asset?
* How would we represent other `PathBatchSpec` types (e.g., `S3BatchSpec`, `AzureBatchSpec`, `GCSBatchSpec`)?

Thank you.